client/src/app/queries/jobfunctions.ts

```diff
@@ -12,13 +17,11 @@ export interface IJobFunctionFetchState {
 export const JobFunctionsQueryKey = "jobfunctions";
 
 export const useFetchJobFunctions = (): IJobFunctionFetchState => {
-  const { data, isLoading, error, refetch } = useQuery(
-    [JobFunctionsQueryKey],
-    async () => (await getJobFunctions()).data,
-    {
-      onError: (error) => console.log("error, ", error),
-    }
-  );
+  const { data, isLoading, error, refetch } = useQuery({
+    queryKey: [JobFunctionsQueryKey],
+    queryFn: getJobFunctions,
+    onError: (error: AxiosError) => console.log("error, ", error),
```

Comment:
Wonder if it would be worth changing all of these to console.error vs console.log. Maybe a separate issue.